notifications.bs

```diff
@@ -647,20 +690,49 @@ getter must return the <a lt="concept notification">notification</a>'s
 getter must return a <a>structured clone</a> of
 <a lt="concept notification">notification</a>'s <a>data</a>.
 
+<p>The <dfn attribute dfn-for=Notification><code>actions</code></dfn> attribute's getter
+must return the <a lt="concept notification">notification</a>'s list of <a>actions</a>.
+
 
 <h3 id=examples>Examples</h3>
 
-<h4 id=using-events>Using events</h4>
+<h4 id=using-events>Using events from a page</h4>
 
-<p>{{Notification}} objects dispatch events during their lifecycle, which
-developers can use to generate desired behaviors.
+<p><a lt="non-persistent notification">Non-persistent</a> {{Notification}}
+objects dispatch events during their lifecycle, which developers can use to
+generate desired behaviors.
 
 <p>The <code>click</code> event dispatches when the user activates a
 notification.
 
 <pre class=example>
-var not = new Notification("Gebrünn Gebrünn by Paul Kalkbrenner", { icon: "newsong.svg", tag: "song" })
-notification.onclick = function() { displaySong(this) }</pre>
+var not = new Notification("Gebrünn Gebrünn by Paul Kalkbrenner", { icon: "newsong.svg", tag: "song" });
```

Comment:
The example used inconsistent variable names, `var not` then `notification.onclick`. That seems worth fixing, though I'm happy to do that as a separate PR if you'd prefer? Left it in for now.